terralab/.terralab-cli-config

```diff
@@ -7,6 +7,7 @@ SERVER_PORT=10444
 # B2C auth config
 OAUTH_OPENID_CONFIGURATION_URI=https://terraprodb2c.b2clogin.com/terraprodb2c.onmicrosoft.com/b2c_1a_signup_signin_tsps_prod/v2.0/.well-known/openid-configuration
 OAUTH_CLIENT_ID=4aefc4ac-d5be-4440-ae9a-aefe4ceb098c
+REMOTE_OAUTH_REDIRECT_URI=https://app.terra.bio/pipelines/cli-auth
```

Comment:
just double checking we dont want this under `services.terra.bio` to feel more scientific service-y vs terra-y

Reply:
i think eventually we will, but this terra-ui PR will go out sooner than we'll be able to set up `services.terra.bio` so we're starting with this and can cut over when `services.terra.bio` is available. i can make a ticket to do that cutover

Reply:
ticket: https://broadworkbench.atlassian.net/browse/TSPS-562